main/level_1.gd

```diff
@@ -2,15 +2,40 @@ extends Node2D
 
 @onready var tilemap: TileMap = get_node("TileMap")
 @onready var camera: Camera2D = get_node("Player/Camera2D")
+@onready var enemy_scene: PackedScene = preload("res://enemies/Enemy.tscn")
+@onready var tile_set: TileSet
+
+var bufferOfMapBorderAndEnemySpawn = 30
```

Comment:
Nice, a good self-descriptive variable

```suggestion
const BUFFER_OF_MAP_BORDER_AND_ENEMY_SPAWN = 30
```

But I will talk to you better about this :)